util/dvsim/SimCfg.py

```diff
@@ -78,6 +78,39 @@ def resolve_dump_format(tool, dump):
     return pick_dump_format(fmts)
 
 
+def expand_run_list(tests, build_map, cfg):
```

Comment:
Why not make this a class method? It will never be called outside of it.

Reply:
> Why not make this a class method? It will never be called outside of it.

I think this can in fact be nested inside `def _create_deploy_objects(self):`.

Reply:
It can be. Should it? Most Python codebases that I've seen avoid heavily nesting functions, since there's not really any benefit and code gets harder to read. If you're worried about information hiding, I could prepend an underscore to the name?

Reply:
Your point about nesting is fair. Should it be a standalone method though?

Reply:
I'll make it a static method of the class. Maybe I've just been writing to much code in non-OO languages lately :-)